jgrapht-core/src/main/java/org/jgrapht/alg/isomorphism/IsomorphismInspector.java

```diff
@@ -30,16 +30,21 @@
 public interface IsomorphismInspector<V, E>
 {
     /**
-     * Get an iterator over all existing (isomorphic) mappings between two graphs.
+     * Get an iterator over all calculated (isomorphic) mappings between two graphs.
      * 
-     * @return an iterator over all existing (isomorphic) mappings between two graphs
+     * @return an iterator over all calculated (isomorphic) mappings between two graphs
+     *
+     * @throws IsomorphismUndecidableException if the isomorphism test was not executed and the inspector cannot decide whether the graphs are isomorphic
      */
     Iterator<GraphMapping<V, E>> getMappings();
 
     /**
      * Check if an isomorphism exists.
-     * 
-     * @return true if there is an isomorphism, false otherwise
+     * An IsomorphismUndecidableException is thrown if the inspector cannot decide whether there is an isomorphism.
```

Comment:
Same here, revert to the old interface and add additional information in the Javadoc of your implementation.